test/execution-tests/3.0.1_resolveModuleName/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+	"compilerOptions": {
+		"noEmitOnError": true,
+		"resolveJsonModule": true
```

Comment:
can we remove the `"resolveJsonModule": true` please? It doesn't do any harm but it might lead someone who was reading the test to believe that it was necessary for `resolveModuleName` to work.  (Not least because of the similarity in naming 😄 )

Reply:
Sure 👍